docs/install/cmd/run.md

```diff
@@ -34,6 +34,12 @@ Secret for validating Docker Hub webhooks.
 
 Can also be set with the `DOCKER_WEBHOOK_SECRET` environment variable.
 
+**--cloudevents-webhook-secret *secret***
```

Comment:
Sorry, this should go alphabetically, so after `**--argocd-namespace *namespace***`